Sunrise/Cells/CartItemCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CartItemCell.swift
+//  Sunrise
+//
+//  Created by Nikola Mladenovic on 5/31/16.
+//  Copyright © 2016 Commercetools. All rights reserved.
+//
+
+import UIKit
+
+class CartItemCell: UITableViewCell {
```

Comment:
Ok, good to know for the future, because I just found the `LineItem` class and was getting confused :D